tests/OmniSharp.Lsp.Tests/OmnisharpPublishDiagnosticFacts.cs

```diff
@@ -44,20 +44,6 @@ private Task ReadyHost(bool roslynAnalyzersEnabled)
                 {{"RoslynExtensionsOptions:EnableAnalyzersSupport", roslynAnalyzersEnabled.ToString()}});
         }
 
-        [Fact]
-        public async Task CheckAllFilesOnNonAnalyzerReturnImmediatlyAllResults()
```

Comment:
Now non analyzer diagnostic worker works the same way as analyzer which uses queue. I think that test is not actual anymore.